docs/markdown/Style-guide.md

```diff
@@ -20,6 +20,14 @@ in Meson build scripts as identifiers for variable.
 
 Let say you would like to refer to your executable so something like `my_exe`.
 
+## Dependency usage
+
+The dependency method is the recommend as a way to handle dependenceis. If your wrap files
```

Comment:
"is the recommend as a way"

Could use some rewording...

Maybe "is the recommended way"?

"dependenceis" -- typo

Reply:
This should say "dependency function" rather than method. And the word dependency should be in backticks so it gets formatter properly. So something like:

    The `dependency` function is

There is also a typo later in the sentence: "dependenceis" instead of "dependencies".

Reply:
Already issues fixed.

Reply:
No it is not. My first comment is still applicable

Reply:
Which part?

Reply:
> "is the recommend as a way"
> 
> Could use some rewording...
> 
> Maybe "is the recommended way"?

Reply:
Oh. Well go and fix that sorry got a lot of stuff to do.

Reply:
Already. Hopefully I covered all of the issues.

Reply:
Thanks.